core/templates/pages/exploration-editor-page/modal-templates/help-modal.component.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for help modal.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { ContextService } from 'services/context.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
+
+@Component({
+  selector: 'oppia-help-modal',
+  templateUrl: './help-modal.component.html',
+})
+export class HelpModalComponent implements OnInit {
+  EDITOR_TUTORIAL_MODE: string;
+  TRANSLATION_TUTORIAL_MODE: string;
+  explorationId: string;
+
+  constructor(
+    private ngbActiveModal: NgbActiveModal,
+    private siteAnalyticsService: SiteAnalyticsService,
+    private contextService: ContextService
+  ) {}
+
+  ngOnInit(): void {
+    this.EDITOR_TUTORIAL_MODE = 'editor';
+    this.TRANSLATION_TUTORIAL_MODE = 'translation';
+    this.explorationId = (
+      this.contextService.getExplorationId());
+  }
+
+  beginEditorTutorial(): void {
+    this.siteAnalyticsService
+      .registerOpenTutorialFromHelpCenterEvent(
+        this.explorationId);
```

Comment:
If the function call fits on one line, let's prefer to write it like this:
```suggestion
    this.siteAnalyticsService.registerOpenTutorialFromHelpCenterEvent(
      this.explorationId);
```

Reply:
(as well as other places)